tests/integration/security/external_ca/main_test.go

```diff
@@ -147,6 +188,7 @@ components:
                 verbs:
                 - approve
 `, map[string]string{"rootcert1": cert1.Rootcert, "signer1": cert1.Signer, "rootcert2": cert2.Rootcert, "signer2": cert2.Signer})
+
 	cfg.ControlPlaneValues = cfgYaml
```

Comment:
When running an external control plane, the base config file for the primary cluster is different than when running normally (with default profile), so IIUC this is using the same overlay over two different bases. I suppose if the tests pass, it's OK, but it seems this would be testing with a (somewhat?) different istiod config from what users would have in an external control plane cluster?

Reply:
It is still using a separate config file for external primary cluster (https://github.com/istio/istio/blob/master/tests/integration/iop-externalistiod-primary-integration-test-defaults.yaml), we are just adding the external cert info on top of that. 
https://github.com/istio/istio/blob/master/pkg/test/framework/components/istio/operator.go#L296
https://github.com/istio/istio/blob/master/pkg/test/framework/components/istio/operator.go#L475